cEP-0020.md

```diff
@@ -0,0 +1,543 @@
+# Newcomer metrics and Gamification
+
+| Metadata |                                                        |
+| -------- | ------------------------------------------------------ |
+| cEP      | 20                                                     |
+| Version  | 1.0                                                    |
+| Title    | Newcomer metrics and Gamification system               |
+| Authors  | Shrikrishna Singh <mailto:krishnasingh.ss30@gmail.com> |
+| Status   | Proposed                                               |
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describes how a gamification system will automate the
+process of becoming a `Developer` from a `Newcomer`, as a part of
+this
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5892040252981248).
+
+For the implementation of this process, this project will
+incorporate a gamification system which will allow us to
+track the progress of newcomers and assign some points to
+each of the activities they do. There will be different
+levels based on point thresholds and a newcomer will also
+get some special privileges after the completion of each
+level. There will also be different badges on the basis of
+performed activities by the newcomers.
+
+So, every newcomer at coala will have to complete the processes
+involved in the gamification system in order to become a developer
+at coala or to get the **coala Developer** badge.
+
+## Background
+
+### Newcomer Experience
```

Comment:
Currently there is no background on the existing newcomer process.

Include a very short summary of it here, and a link to http://coala.io/newcomer for readers to see the current version, *and* a link to the github latest commit for the document, to archive what it looked like when the cEP was created.

Then the problems with it (already below) will make more sense to the average reader.

Here would be a good spot to explain that the coala Developer role is implemented as a GitHub organisation team. (and that should be the only time 'team' is mentioned in the cEP.)